client/post-editor/post-editor.jsx

```diff
@@ -592,10 +602,13 @@ export const PostEditor = React.createClass( {
 
 		previewPost = function() {
 			if ( this._previewWindow ) {
-				this._previewWindow.location = this.state.previewUrl;
+				this._previewWindow.location = ( this.state.previewAction === 'view' ? this.state.post.URL : this.state.previewUrl );
 				this._previewWindow.focus();
 			} else {
-				this._previewWindow = window.open( this.state.previewUrl, 'WordPress.com Post Preview' );
+				this._previewWindow = window.open(
+					( this.state.previewAction === 'view' ? this.state.post.URL : this.state.previewUrl ),
```

Comment:
Since this ternary expression occurs four separate times, maybe it'd be worth moving to a helper function on the component?

```js
getPreviewUrl() {
	const { previewAction, previewUrl, post } = this.state;
	if ( 'view' === previewAction ) {
		return post.URL;
	}

	return previewUrl;
}
```